app/models/request.rb

```diff
@@ -87,6 +87,9 @@ def date_for_state(state)
   # reflect the final product of a request.
   has_many :related_aliquots, class_name: 'Aliquot', inverse_of: :request
 
+  # Can have many key value pairs of metadata
+  has_many :poly_metadata, as: :metadatable
```

Comment:
Think it makes sense to add `dependent: destroy` here, so if a request is destroyed, so is its metadata.

Reply:
Yes done. Had it in the v2 resource but maybe that's wrong.
What about `, inverse_of: :metadatable`? can't work out if that's needed, or does anything

Reply:
RE `inverse_of` - I wondered this too. Docs here - https://guides.rubyonrails.org/association_basics.html#bi-directional-associations

Think it's unnecessary unless you use `:through` or `:foreign_key`

> Active Record supports automatic identification for most associations with standard names. However, bi-directional associations that contain the :through or :foreign_key options will not be automatically identified.